packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -51,6 +121,17 @@ export default function QueueListHeader({
           </NotesText>
         </Container>
       )}
+      <div style={{ paddingLeft: "64px" }}>
+        {queue?.allowQuestions ? (
+          <NotesText style={{ color: "green" }}>
+            This queue is allowing new questions
```

Comment:
Since students probably assume the queue is allowing new questions we might only need to show the text when it's **not** allowing new questions. 

Either way works tho